lib/fluent/plugin/monitoring.rb

```diff
@@ -13,9 +13,27 @@
 # limitations under the License.
 
 module Monitoring
+  # Base class for the counter.
+  class BaseCounter
+    def increment(_labels, _count)
```

Comment:
Copying the comment from the other PR here in case we want to merge this one on its own:

`_labels, _count` => `_by: 1, _labels: {}`?

Reply:
Changed from mobile while OOO in order to unblock the release. I hope that works.